react/queries/getRouteRedemptionCode.gql

```diff
@@ -0,0 +1,3 @@
+query getRouteRedemptionCode {
+  getRouteRedemptionCode
```

Comment:
Hi! I saw that only the updateGiftCard mutation has the provider param, isnt that necessary for all other queries?

Reply:
No, the other querys dont need to send a param, because this only a get, so I dont need to send any param